Dockerfile-mongo-seed

```diff
@@ -0,0 +1,13 @@
+FROM  mongo:3.2
+
+COPY dc-campaign-finance-mongodatabase.zip .
+
+# update packages and intall unzip
+RUN apt-get update
+RUN apt-get install unzip
+
+# unzip db .bson files to root dir
+RUN unzip dc-campaign-finance-mongodatabase.zip -d .
+
+# restore mongodb using docker compose name
+CMD mongorestore --host mongodb -d dc-campaign-finance --drop ./dc-campaign-finance-mongodatabase/dc-campaign-finance
```

Comment:
You might try and redo this in exec form, not in shell form. Docker suggests that you use this form as much as possible. Also you might look at using ENTRYPOINT and then using the last CMD to pass the arguments to it? There is a plethora of information about these commands here: http://stackoverflow.com/questions/21553353/what-is-the-difference-between-cmd-and-entrypoint-in-a-dockerfile